s4e-backend/src/main/java/pl/cyfronet/s4e/config/SecurityConfig.java

```diff
@@ -81,6 +81,8 @@ private void authorizeRequests(ExpressionUrlAuthorizationConfigurer<HttpSecurity
 
                 .mvcMatchers(GET, prefix("/jwt/pubkey")).permitAll()
 
+                .mvcMatchers(DELETE, prefix("/invitation/{token}"))
```

Comment:
Shouldn't it be /invitation**s**/{token}?

Reply:
Also, another thing is that you don't have the institution in the path, so you cannot reference it in the access rule. Either modify the endpoint to be prefixed with `/instititutions/{slug}/` (it would then fall under existing rule) or extend ISH with a checking method such as `isInvitationAdmin` or sth like this. Or `canDeleteInvitation(#token)`, this would hint at entity access model - I would create another class for that, which may later be extended to encompass other entities.